src/EcomProvider.cs

```diff
@@ -300,6 +320,12 @@ public override Schema GetOriginalSourceSchema()
         {
             result.RemoveTable(table);
         }
+        return result;
+    }
+
+    public override Schema GetOriginalSourceSchema()
```

Comment:
all contnent can be moved to the DestinationSchema except:
EcomProductCategoryFieldValue.FieldValueProductNumber table as it is used in both

Reply:
it was mainly because of "DiscountAccessUser" & "DiscountAccessUserGroup" that i was splitting the schema into the Source and the Destination.. so now i kept everything as before, but moved these two to the destination as they are not used in the Source

Reply:
ok, now it looks aligned :)